modules/custom/d_p_tiles/js/d_p_tiles.js

```diff
@@ -9,9 +9,25 @@
 
   Drupal.behaviors.d_p_tiles = {
     attach: function (context, settings) {
+
       // Enable Masonry.
       $(document).ready(function() {
         var $wrapper = $('.d-tiles, .d-side-tiles', context);
+        var videoElements = $wrapper.find('.video-embed');
+
+        window.onresize = function(event) {
```

Comment:
Pytanie, czy embedy nie wariują przy intensywnym zmienianiu rozmiaru i przy przesuwaniu palcem na telefonie - bo wtedy onresize jest wywoływane milion razy. Bo jeśli tak się dzieje to można zmienić na coś takiego:

```js
function doit(){
    // Rob costam.
}

var timer;
window.onresize = function(){
  clearTimeout(timer);
  timer = setTimeout(doit, 100);
};```